package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Simple search API to query the SFR Elasticsearch index.",
   "author": "NYPL Digital",
   "license": "MIT",
-  "version": "0.2.0",
+  "version": "0.2.2",
```

Comment:
0 -> 2 ?

Reply:
Forgot to bump the version here after the last update :/